src/docs/asciidoc/index.adoc

```diff
@@ -62,25 +63,61 @@ NOTE: Please also consult the official Spring Boot documentation: https://docs.s
         <relativePath/> <!-- lookup parent from repository -->
     </parent>
 
+    <properties>
+        <joinfaces.version>{revnumber}</joinfaces.version>
+    </properties>
+
     <dependencyManagement>
         <dependencies>
             <dependency>
                 <groupId>org.joinfaces</groupId>
                 <artifactId>joinfaces-dependencies</artifactId>
-                <version>{project-version}</version>
+                <version>${joinfaces.version}</version>
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
         </dependencies>
     </dependencyManagement>
 
     <dependencies>
+        <dependency>
+            <groupId>org.joinfaces</groupId>
+            <artifactId>jsf-spring-boot-starter</artifactId>
+        </dependency>
+
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.joinfaces</groupId>
+                    <artifactId>joinfaces-maven-plugin</artifactId>
+                    <version>${joinfaces.version}</version>
+                    <executions>
+                        <execution>
+                            <goals>
+                                <goal>classpath-scan</goal>
```

Comment:
Is it necessary to manually specify the execution of the classpath-scan goal?

Reply:
Yes. I think it is common maven usage. However, it is not necessary to inform the maven execution phase.